karyon3-core/src/main/java/com/netflix/karyon/KaryonBuilder.java

```diff
@@ -0,0 +1,79 @@
+package com.netflix.karyon;
+
+import com.netflix.governator.DefaultGovernatorConfiguration;
+import com.netflix.governator.Governator;
+import com.netflix.governator.LifecycleInjector;
+import com.netflix.governator.auto.ModuleListProviders;
+import com.netflix.governator.auto.PropertySource;
+
+/**
+ * Extension to Governator's {@link DefaultGovernatorConfiguration.Builder} for specific
+ * use in Karyon with the following added functionality,
+ * 
+ * 1.  Capture a sequence of calls to the Builder using zero or more {@link KaryonSuite}'s
+ * 2.  start() method to create the actual injector as a shortcut to calling
+ *          Goverantor.createInjector(builder.build())
+ * 3.  startAndAwaitTermination() to create and block on the application lifecycle as a shortcut
+ *     to calling
+ *          Goverantor.createInjector(builder.build()).startAndAwaitTermination()
+ * 
+ * @author elandau
+ *
+ */
+public class KaryonBuilder extends DefaultGovernatorConfiguration.Builder<KaryonBuilder> {
```

Comment:
I guess this is my confusion comes from. The KaryonBuilder is extended from a builder as part of governator configuration? I think the relationship should be composition instead of extension.